mani_skill2/envs/sapien_env.py

```diff
@@ -969,9 +972,6 @@ def render_human(self):
                 self._viewer.set_camera_pose(
                     self._human_render_cameras["render_camera"].camera.global_pose
                 )
-
-        for obj in self._hidden_objects:
-            obj.show_visual()
```

Comment:
oh is this because by default it is always shown except for rendering observations? I think we should do this. The default should be it is always hidden. Only when render_human is called is it shown.

Reply:
Well to be honest it may not matter, we don't call any of the expensive operations if the object is already hidden

Reply:
actually ignore this. re-read ur comments